.github/workflows/workflow.yml

```
@@ -6,8 +6,8 @@ on:
   push:
   pull_request:
   schedule:
-    # every day at 13:00 UTC excluding Saturday and Sunday
-    - cron: "0 13 * * 1-5"
+    # every Tuesday (2) at 15:00 and 18:00 UTC / UTC is 1h before us.
```

This depends on daylight saving 
```suggestion
    # every Tuesday (2) at 15:00 and 18:00 UTC
```